students/unsan-Jeong/users/views.py

```diff
@@ -0,0 +1,34 @@
+import json ,re
+
+from django.http     import JsonResponse, HttpResponse
+from django.views    import View
+
+#from django.core.exceptions import ValidationError
+
+from users.models import Users
+
+class SignUp(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if len(data['password']) < 8:
+                return JsonResponse({'MESSAGE':'password of at least eight characters'}, status=404)
+            if (re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', data['email']) == None):
+                return JsonResponse({'MESSAGE':'Invalid email'}, status=400)
+            if Users.objects.filter(name=data['name']).exists():
+                return JsonResponse({'MESSAGE':'Name already exists'}, status=400)
+            if Users.objects.filter(phone_number=data['phone_number']).exists():
+                return JsonResponse({'MESSAGE':'Phone_number already exists'}, status=400)
+            if Users.objects.filter(email=data['email']).exists():
+                return JsonResponse({'MESSAGE':'email already exists'}, status=400)
```

Comment:
```suggestion
            if (re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', data['email']) == None):
                return JsonResponse({'MESSAGE':'Invalid email'}, status=400)
                
            if Users.objects.filter(name=data['name']).exists():
                return JsonResponse({'MESSAGE':'Name already exists'}, status=400)
                
            if Users.objects.filter(phone_number=data['phone_number']).exists():
                return JsonResponse({'MESSAGE':'Phone_number already exists'}, status=400)
                
            if Users.objects.filter(email=data['email']).exists():
                return JsonResponse({'MESSAGE':'email already exists'}, status=400)
                
```
이게 조금 더 가독성 있지 않을까요?